dsn.go

```diff
@@ -465,6 +470,18 @@ func parseDSNParams(cfg *Config, params string) (err error) {
 				return errors.New("invalid bool value: " + value)
 			}
 
+		// allow resultset metadata being optional
+		case "resultSetMetadata":
```

Comment:
As a mysql variable `resultset_metadata` already exists, it would be better to use it (like `time_zone`).